main.go

```diff
@@ -271,6 +276,15 @@ func ShowVersion() {
 	os.Stdout.Write([]byte(v))
 }
 
+func PrintArtifactName() {
+	aname, err := ioutil.ReadFile("/etc/mender/artifact_info")
+	if err != nil {
+		fmt.Printf("failed to read artifact name: err: %v", err)
+		return
+	}
+	fmt.Println(aname)
```

Comment:
This is the format of the file:

```
artifact_name=mender-image-1.0
```

It should only print the value, not the key. And you should assume that there may be other fields in there in the future.